packages/storybook/stories/CallComposite/snippets/Container.snippet.tsx

```diff
@@ -22,6 +23,16 @@ export type ContainerProps = {
 };
 
 const isTeamsMeetingLink = (link: string): boolean => link.startsWith('https://teams.microsoft.com/l/meetup-join');
+const isGroupLink = (link: string): boolean => link.indexOf('-') !== -1;
```

Comment:
link.indexOf('-') !== -1; seems like a little hacky for our customer to check (this snippet would be a public one exposed to our customer, they will have to know the syntax difference from groupId, roomId), is there a way to check it like a function isGuid()?

Reply:
Good point. Will polish that in this bug https://skype.visualstudio.com/SPOOL/_workitems/edit/2925571. Thanks.